src/release/breaking-changes/dispose.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Added missed `dispose()` for some disposable objects in Flutter
+description: >
+  'dispose()' may fail because of double disposal.
+---
+
+## Summary
+
+Missed 'dispose()' are added for some disposable objects. If some other code
+also invokes 'dispose()' for the objects, and the object is protected from double
+disposal, the second 'dispose()' will fail with error message:
+
+`Once you have called dispose() on a <class name>, it can no longer be used.` 
+
+## Background
+
+There is a rule that owner of an object should dispose it.
+
+This rule was broken in some places in Flutter Framework: owners were not disposing the disposable objects.
+The issue was fixed by adding `dispose()`. The added `dispose()` can cause failures in debug mode,
+if there is another `dispose()`, and the object is protected from double disposal.
```

Comment:
**chore (if-minor):** Use [semantic breaks](https://sembr.org/). Condition, then instruction per [GSG](https://developers.google.com/style/sentence-structure). Write in active voice per [GSG](https://developers.google.com/style/voice).
```suggestion
This rule was broken in some places in Flutter Framework:
owners were not disposing the disposable objects.
To fix the issue, `dispose()` was added.
If there is another `dispose()`, and the object is protected from double disposal,
the added `dispose()` can cause failures in debug mode.
```

Reply:
Please fix the grammar:

=>
This convention was broken in some places:
owners were not disposing the disposable objects.
The issue was fixed by adding a call to `dispose()`.
However, if the object is protected from double disposal,
this can cause failures when running in debug mode
and `dispose()` is called elsewhere on the object.